md/docs/derby-0.6/models/backends.md

```diff
@@ -1,38 +1,43 @@
 # Backends
 
-Racer stores are backed by ShareJS, which uses [LiveDB](https://github.com/share/livedb) to persist data, perform queries, keep a journal of all operations, and pub/sub operations and changes to queries. Currently, LiveDB has adapters for in memory or Redis based pub/sub and supports storage in memory or MongoDB via [livedb-mongo](https://github.com/share/livedb-mongo). LiveDB is written with support for additional database adapters in mind.
+Racer stores are backed by ShareDB, which is used to persist data, perform queries, keep a journal of all operations, and pub/sub operations and changes to queries. Currently, ShareDB has [two pub/sub adapters](https://github.com/share/sharedb#database-adapters): one for in memory and one for Redis based pub/sub. ShareDB supports in memory or MongoDB storage. The database adapter [ShareDBMongo](https://github.com/share/sharedb-mongo)
+is backed by a real Mongo database and full query support. ShareDB is written with support for additional database adapters in mind.
 
 Getting started with a single-process server and MongoDB:
 
 ```js
-var derby = require('derby');
-var livedb = require('livedb');
-var liveDbMongo = require('livedb-mongo');
+var racer = require('racer');
+var ShareDbMongo = require('sharedb-mongo');
 
-var db = liveDbMongo('mongodb://localhost:27017/test');
-var backend = livedb.client(db);
-var store = derby.createStore({backend: backend});
+var db = new ShareDbMongo({mongo: 'mongodb://localhost:27017/test'});
```

Comment:
`var db = new ShareDbMongo('mongodb://localhost:27017/test');` is the shorter form also supported